lib/validate_test.go

```diff
@@ -0,0 +1,87 @@
+package samplify
```

Comment:
Try writing your tests in a separate _test package. samplify_test in this case. Gives you an idea of how will your package and the functions in the package are used by other packages.